lib/logstash/inputs/logstash.rb

```diff
@@ -1,5 +1,12 @@
 # encoding: utf-8
 
+########################################################################
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V.
+# under one or more contributor license agreements. Licensed under the
+# Elastic License 2.0; you may not use this file except in compliance
+# with the Elastic License 2.0.
+########################################################################
+
```

Comment:
This project is has the `Apache-2.0` license and should NOT have Elastic-license headers.

```suggestion
```